### PR TITLE
[WIP] Docs for packaging as Docker image.

### PR DESCRIPTION
**Pull request by treeder**
For review.

### CODE REVIEW

**Comment by treeder** [comment]
feedback requested @arschles @rdallman @rkononov

File: ruby/README.md [hunk 1]
```diff
@@ -16,58 +16,109 @@ Notice in `hello.rb`, we add the following so it uses the vendored gems:
...
-Follow the same steps above to run/test your worker. But instead of zipping it up, do the following:
+## Private images
 
-Package it up inside an image and send it off to Docker HUB, see the `Dockerfile` for reference:
+If you want to keep your code private and use a [private Docker repository](https://docs.docker.com/docker-hub/repos/#private-repositories), you just need
+to let Iron know it's private.
 
 ```sh
-docker build -t treeder/hello.rb:latest .
+iron docker login -e YOUR_DOCKER_HUB_EMAIL -p YOUR_DOCKER_HUB_PASSWORD
```
Comment (arschles) [inline ruby/README.md]: should leave a note here about how we store these credentials on our machines and what guarantees we make about their fidelity

**Comment by rkononov** [comment]
lgtm, but need to add commands to ironcli before merging

**Comment by rdallman** [comment]
ditto

**Comment by rkononov** [comment]
@treeder could we merge this one?

**Comment by treeder** [comment]
Let me go through it and make sure everything works first.

**Comment by treeder** [comment]
Went through it all and updated a bit.  All works!  Should apply this to all languages.

**Comment by rdallman** [comment]
lgtm

**Pull request merged**